src/model.ts

```diff
@@ -168,6 +177,9 @@ export class Model {
         case 'minbatchsize':
           minbatchsize = parseInt(obj.toString(), 10);
           break;
+        case 'minbatchtimeout':
+          minbatchtimeout = parseInt(obj.toString(), 10);
```

Comment:
why 10?

Reply:
because it's a decimal number (and the default radix is not always 10, for example, it'll default to 8 for strings that start with 0).